README.md

```diff
@@ -624,6 +622,19 @@ WriteRequest{
 
 ### Privacy Considerations
 
+The `did:earth` method is a fully compliant IID and DID. As such the privacy considerations taken into account are applicable to this method. The reader is directed towards the privacy considerations discussed in [section 10 of the W3C DID Core specification](https://w3c.github.io/did-core/#privacy-considerations).
+
+#### Linked Resources
+
+Linked resources within IIDs support privacy in three key three key ways:
+
+1.  Avoids posting potentially sensitive information on-chain in an unavoidably public and irrevocable manner.
+2.  Provides a service endpoint that can apply appropriate privacy and security checks before revealing information.
+3.  The hashgraph resource descriptor type obscures not only the content of the linked resource, but also the quantity of resource objects.
+
+
```

Comment:
We can replace this whole section with something that captures the points in my previous comment. We probably don't want literal bullet points, although an introductory set of bullet titles followed by a paragraph for each bullet should work well.